modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -221,6 +166,7 @@ where:
 [NOTE]
 ====
 If your cluster uses Technology Preview features that are enabled by the `TechPreviewNoUpgrade` feature set, you must include the `--enable-tech-preview` parameter.
+//Does --included make this obsolete?
```

Comment:
Yes, I expect `ccoctl`'s `--enable-tech-preview` is redundant now that `--included` will perform that aspect of filtering (along with the other aspects like cloud provider, enabled cluster capabilities, etc.).

Reply:
--enable-tech-preview option is a must for ccoctl to handle TP CR. So I think "--enable-tech-preview" option shall be always added for "ccoctl create" command.

Reply:
@jeana-redhat PTAL at this comment. "--enable-tech-preview" shall not be removed, instead it is a must after having "--include". Unless ccoctl makes code change of removing "--enable-tech-preview".

Reply:
Thanks for confirming that!